app/components/CalendarPage.js

```diff
@@ -0,0 +1,39 @@
+import React, { PropTypes } from 'react';
+import { getMonth, getDay, getHours, getMinutes } from '../utils/unifiedDateFormat';
+
+function CalendarPage({ date, className, showTime }) {
+    return (
+        <div className={`${className}`}>
```

Comment:
@AlexeyBarkow why don't you use bootstrap layout as much as possible? Don't invent bicycle :)